CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## TBD
+
+### Enhancements
+
+* Max reported threads can now be configured via the manifest.
```

Comment:
```suggestion
* Max reported threads can now be configured using manifest meta-data "com.bugsnag.android. MAX_REPORTED_THREADS"
```

Maybe? Just so that it's super clear?